### PR TITLE
[Elixir] Add analyzer comments for boutique-inventory

### PR DESCRIPTION
**Pull request by fmmatheus**
Corresponding analyzer PR: https://github.com/exercism/elixir-analyzer/pull/194

### CODE REVIEW

File: analyzer-comments/elixir/boutique-inventory/use_enum_filter_or_enum_reject.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Use `Enum.filter` or `Enum.reject`
+
+One purpose of this exercise it to learn about filters. Use the `Enum.filter` or `Enum.reject` function in the `with_missing_price` function to practice.
```
Comment (angelikatyborska) [inline analyzer-comments/elixir/boutique-inventory/use_enum_filter_or_enum_reject.md:3]:
```suggestion
Use the `Enum.filter` or `Enum.reject` function in the `with_missing_price` function. Those are the best fitting `Enum` functions for this task.
```

File: analyzer-comments/elixir/boutique-inventory/use_enum_map.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Use `Enum.map`
+
+One purpose of this exercise it to learn about map. Use the `Enum.map` function in the `increase_quantity` function to practice.
```
Comment (angelikatyborska) [inline analyzer-comments/elixir/boutique-inventory/use_enum_map.md:3]:
```suggestion
Use the `Enum.map` function in the `increase_quantity` function to practice. It's the best fitting `Enum` function for this task.
```

File: analyzer-comments/elixir/boutique-inventory/use_enum_sort_by.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# Use `Enum.sort_by`
+
+One purpose of this exercise it to learn about sort_by. Use the `Enum.sort_by` function in the `sort_by_price` function to practice.
```
Comment (angelikatyborska) [inline analyzer-comments/elixir/boutique-inventory/use_enum_sort_by.md:3]:
```suggestion
Use the `Enum.sort_by` function in the `sort_by_price` function to practice. It's the best fitting `Enum` function for this task.
```

File: analyzer-comments/elixir/boutique-inventory/use_enum_reduce.md [hunk 4]
```diff
@@ -0,0 +1,3 @@
+# Use `Enum.reduce`
+
+One purpose of this exercise it to learn about reduce. Use the `Enum.reduce` function in the `total_quantity` function to practice.
```
Comment (angelikatyborska) [inline analyzer-comments/elixir/boutique-inventory/use_enum_reduce.md:3]:
```suggestion
Use the `Enum.reduce` function in the `total_quantity` function to practice. It's the best fitting `Enum` function for this task.
```

**Review by angelikatyborska — COMMENTED** [review]
Saying that the exercise is explicitly about learning filtering or learning reduce is not entirely what I had in mind here 🤔. I guess it's hard to find the correct words here. It has to fit a solution where somebody didn't use Enum at all, and when somebody used the wrong Enum function (e.g. reduce when map is better).

I'm suggesting a rephrasing for all of the comments (the same everywhere). What do you think?

File: analyzer-comments/elixir/boutique-inventory/use_enum_map.md [hunk 5]
```diff
@@ -0,0 +1,3 @@
+# Use `Enum.map`
+
+Use the `Enum.map` function in the `increase_quantity` function to practice. It's the best fitting `Enum` function for this task.
```
[the remark this answers is not in the document]
Comment (jiegillet) [reply analyzer-comments/elixir/boutique-inventory/use_enum_map.md:3]: Yes, let's do this one in the other PR

**Review by angelikatyborska — APPROVED** [review]

**Pull request merged**